app/observers/response_observer.rb

```diff
@@ -5,5 +5,6 @@ class ResponseObserver < ActiveRecord::Observer
 
   def after_create(response)
     response.prediction.response_notifications.each(&:new_activity!)
+    TagAdder.new(prediction: response.prediction, string: response.comment).call
```

Comment:
So when I gave this a test run, I found it confusing that this worked in responses and not in the original prediction text. Why wouldn't we add it to the prediction text also? Seems trivial to do that as well with this `TagAdder` class already written.

Reply:
Michael, as you say, this could easily be adapted to the original prediction description as well as responses. One thing I didn't want to do is have the predictions index get cluttered with hash tags or for the tags to count against its enforced description length.

If we are happy with the overall approach here, I don't mind doing some additional work to handle that case. Perhaps we could setup the associations when we build new predictions, but strip out the hash tags that were originally part of the description before saving.

Does that make sense?

Reply:
That's a good point. If that's the case, can we just add a little bit of copy to the predictions show page that tells people about the functionality? Like "You can now add hashtags to predictions. Just include them in your response (e.g. #my-tag)"?

Reply:
I think as long as there's an explanation there we can leave the functionality as is

Reply:
I added some copy explaining how the hashtags work [here](https://github.com/bellroy/predictionbook/pull/201/commits/d627926d3e7892ebdef100cda5740e04416f1515), but I plan on a followup PR that'll allow hashtags in the prediction description (likely after Christmas).

👍🏼